libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -800,3 +807,13 @@ void AP_MotorsMulticopter::convert_pwm_min_max_param(int16_t radio_min, int16_t
     _pwm_min.set_and_save(radio_min);
     _pwm_max.set_and_save(radio_max);
 }
+
+bool AP_MotorsMulticopter::check_failsafe()
```

Comment:
let's put a comment above the method implementation as well.

Reply:
Fixed